plugin-api/build.gradle

```diff
@@ -1,6 +1,8 @@
 apply plugin: 'com.github.johnrengelman.shadow'
 
 dependencies {
+  api libs.slf4j
```

Comment:
it doesn't matter that much because we don't use the new APIs in our wrappers, but could be a good chance to upgrade to slf4j 2?

Reply:
That's also in my next plans. That would require products to update to slf4j 2 when they will update to this new API version. I checked, and it is already the case in [SQ](https://github.com/SonarSource/sonar-enterprise/blob/d72ce6b2e755bb7b626add3dc19dbae49cf871cb/build.gradle#L335), but not yet in [SC](https://github.com/SonarSource/sonarcloud-core/blob/e9913df7419c72e79c30c9b8a3118cc0c537ea6d/build.gradle#L341) or SL. I have created a [ticket](https://sonarsource.atlassian.net/browse/SLCORE-455) in SL, and I will ask SC to do the same.

Reply:
makes sense, thanks